### PR TITLE
LICENSE update: Add MIT license for infrastructure / scripts

### PR DESCRIPTION
**Pull request by bebatut**
Someone asked lately if he could reuse the infrastructure and scripts. We realized that the current LICENSE (CC-BY) is not covering that.

So here I changed the LICENSE file to MIT and made sure that CC-BY 4.0 (for content) is mentioned on the README, home page, FAQ.

Hopefully we can get the explicit approval of all the contributors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,9 @@ If you want to build the website locally, please have a look at the [tutorial](h
 
 # License
 
-This work is licensed under the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0).
+This content of this work is licensed under the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0).
```
Comment (erasche) [inline README.md]:
```suggestion
The content of the tutorials and website are licensed under the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0).
```

maybe? Just want to be a bit more explicit.

**Comment by nsoranzo** [comment]
I think that with this change we also need to modify the tutorial footer and last page of the slide decks to include a CC BY 4.0 reference.

**Comment by bebatut** [comment]
As suggested by @nsoranzo:

Screenshot 2019-09-17 at 15 55 08


Screenshot 2019-09-17 at 15 55 51

**Comment by nsoranzo** [comment]
Thanks @bebatut ! Should the sentence also be in `_includes/default-footer.html` ? Then it could be removed from `_layouts/home.html`

**Comment by bebatut** [comment]
As suggested by @nsoranzo, I put the sentence in the footer for all pages and remove the mention on the home page

Home (topic pages will have the same footer)

Screenshot 2019-09-18 at 09 33 07

Tutorial

Screenshot 2019-09-18 at 09 33 14

**Comment by bernt-matthias** [comment]
Looks good to me.. but I'm no expert with licences at all...

**Comment by erasche** [comment]
We're still waiting for approvals from the following people, if you have time to approve this it would be super great! We're just changing the license on the website infrastructure (layout, scripts, etc) to be MIT rather than CC-BY since that wasn't so appropriate. Thanks so much!

- @bagnacan 
- @bedroesb 
- @nekrut 
- @Delphine-L 
- @frederikcoppens 
- @jmchilton 
- @lecorguille 
- @martenson 
- @mtekman 
- @mwolfien 
- @njall 
- @Slugger70 
- @tobyhodges 
- @tnabtaf 
- @willdurand

**Comment by tobyhodges** [comment]
👍

File: LICENSE.md [hunk 2]
```diff
@@ -1 +1,21 @@
-This work is licensed under the Creative Commons Attribution 4.0 International License. To view a copy of this license, visit https://creativecommons.org/licenses/by/4.0/ or send a letter to Creative Commons, PO Box 1866, Mountain View, CA 94042, USA.
+The MIT License (MIT)
```
Comment (willdurand) [inline LICENSE.md]: it should be possible to have two license files, I guess.

File: LICENSE.md [hunk 3]
```diff
@@ -1 +1,21 @@
...
+
+Copyright (c) 2019 Galaxy Training Network
```
Comment (willdurand) [inline LICENSE.md]: I suggest removing the date here.

**Comment by frederikcoppens** [comment]
👍

**Comment by mtekman** [comment]
:+1:

**Comment by jmchilton** [comment]
👍

**Comment by njall** [comment]
(y)

**Comment by Delphine-L** [comment]
👍

**Comment by abotzki** [comment]
we are very happy to see that already a majority has approved the proposal for MIT as license for the underlying infrastructure. Since we are very much interested to use the infrastructure for our training material I was hoping that we could go ahead deploying them soon. Thanks.

**Comment by bgruening** [comment]
I contacted the missing people via mail.

**Comment by bagnacan** [comment]
+1
Go for it

**Comment by hexylena** [comment]
I received out of band approval from @tnabtaf. 

This *should* constitute everyone who has made changes to the infrastructure portion according to git contribution logs. If I've missed anyone, I hope you'll forgive us :pray: 

With that, we can finally declare **the training materials infrastructure are MIT licensed!** :tada::100::tada::fireworks::hugs::birthday::balloon:🎓✨

Thank you to all of our awesome community for the contributions over the years, and here's to a hopefully brighter future with a more sensible license allowing more organisations to adopt the GTN's hard work.

**Pull request merged**